cat-launcher/src-tauri/src/manual_backups/manual_backups.rs

```diff
@@ -0,0 +1,160 @@
+use std::path::{Path, PathBuf};
+
+use crate::filesystem::paths::{
+    get_or_create_manual_backup_archive_filepath, get_or_create_user_game_data_dir,
+    GetManualBackupArchivePathError, GetUserGameDataDirError,
+};
+use crate::infra::archive::{create_zip_archive, extract_archive, ExtractionError};
+use crate::infra::utils::OS;
+use crate::manual_backups::repository::{ManualBackupRepository, ManualBackupRepositoryError};
+use crate::variants::GameVariant;
+
+#[derive(thiserror::Error, Debug)]
+pub enum ListManualBackupsError {
+    #[error("failed to get backup entries: {0}")]
+    Get(#[from] ManualBackupRepositoryError),
+}
+
+pub async fn list_manual_backups(
+    game_variant: &GameVariant,
+    backup_repository: &impl ManualBackupRepository,
+) -> Result<Vec<crate::manual_backups::repository::ManualBackupEntry>, ListManualBackupsError> {
+    let backups = backup_repository
+        .get_manual_backups_sorted_by_timestamp(game_variant)
+        .await?;
+    Ok(backups)
+}
+
+use crate::infra::archive::ArchiveCreationError;
```

Comment:
Move all use statements to the top of the file in this order: stdlib, third-party lib, crate and super.